tls/extensions/s2n_client_key_share.c

```diff
@@ -294,6 +299,25 @@ static int s2n_client_key_share_recv_ecc(struct s2n_connection *conn, struct s2n
     return S2N_SUCCESS;
 }
 
+static int s2n_client_key_share_recv_hybrid_partial_ecc(struct s2n_stuffer *key_share, struct s2n_kem_group_params *new_client_params)
+{
+    const struct s2n_kem_group *kem_group = new_client_params->kem_group;
+
```

Comment:
Don't forget to check what you're going to dereference
```suggestion
{
    POSIX_ENSURE_ref(new_client_params);
    const struct s2n_kem_group *kem_group = new_client_params->kem_group;
    POSIX_ENSURE_REF(kem_group);

```

Reply:
Fixed